release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1923,6 +1906,24 @@ In this update, `systemd` service only sets the default RPS mask for virtual int
 
 * Before this update, some of the sidebar items did not contain namespace context. Consequently, when links were opened from another browser, or opening links from a different active namespace, the web console does not switch to the correct namespace. With this update, the correct namespace is selected when opening the URL. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2039647[*BZ#2039647*])
 
+* Previously, when using the console to instantiate a template, its parameters were stored as a secret resource. When the template was removed, the secret remained. This consequently created an unnecessary buildup of secrets in a cluster. With this update, an ownership reference is added to the secret that maps to the template instance. Now, when the template instance is removed, the secret is also removed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2015042[*BZ#2015042*])
+
+* With this update, the `jsonData` property has been deprecated and replaced with `data` in the `ping` source. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2084438[*BZ#2084438*])
+
+* Previously, the topology view in the {product-title} web console would fail or lag for clusters with more than 100 nodes. With this update, the topology view shows a `LimitExceeded` state for clusters with more than 100 nodes. An option is provided to view the resources by using the *Search* page instead. Alternatively, you can click Show topology anyway to continue to load the topology view. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2060329[*BZ#2060329*])
+
+* Previously, if a service exposed multiple service ports and the route target port was `8080`, attempts to change the target port would result in another service port being updated instead of the port `8080` service port. With this update, the service port corresponding to the active target port is replaced when a new target port is set. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2077943[*BZ#2077943*])
+
+* Previously, the `git` detection used to manage instance APIs to get repository information did not work for repositories from self hosted GitHub and Bitbucket. With this update, detection for self hosted GitHub and Bitbucket instance repositories works. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2038244[*BZ#2038244*])
+
+* Previously, `apiVersion` was not passed in the correct format to *Resource* drop-down menu in the *EventSource* creation form. This caused `InContext` to not be selected under *EventSource* creation, which excluded it from the *Resource* drop-down menu. With this update, the *Resource* drop-down menu includes `Resource` from `InContext`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2070020[*BZ#2070020*])
```

Comment:
You could update "the correct format to *Resource* drop-down menu" to "the correct format to the *Resource* drop-down menu".